torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -63,39 +63,30 @@ IValue representativeValue(Value* v) {
 
 void PropagateShapeOnBlock(Block * block, bool insert_expands=true);
 
-// for each node in the schema with type Tensor, extract the TensorType
+// for each node in the schema with type Tensor, extract the CompleteTensorType
 // returns at::nullopt if any Tensor in the schema does not have a known shape
 // ignores non-tensor in the list of inputs
```

Comment:
Isn't this out of date now?

Reply:
It's slightly out of date. `CompleteTensorType` should change to the name of the template parameter, but apart from that it looks good.